boards/Kconfig

```
@@ -370,7 +370,16 @@ config ARCH_BOARD_ESP32S2_SAOLA_1
 	select ARCH_HAVE_IRQBUTTONS if ESP32S2_GPIO_IRQ
 	---help---
 		This is the ESP32-S2-Saola-1 board
-
+		
+config ARCH_BOARD_ESP32S2_FRANZININHO
+	bool "Espressif ESP32-S2-FRANZININHO"
```

```suggestion
	bool “Franzininho Wi-Fi"
```
The correct name for this board is Franzininho Wi-Fi.
Furthermore, this board is not an official design from Espressif, so “Espressif” should not appear in the board name.